packages/office-addin-lint/src/lint.ts

```diff
@@ -43,13 +47,17 @@ export function getLintFixCommand(files: string): string {
 }
 
 export function performLintFix(files: string) {
-  try{
+  try {
     const command = getLintFixCommand(files);
     execCommand(command);
     usageDataObject.sendUsageDataSuccessEvent("performLintFix");
   } catch (err) {
-    usageDataObject.sendUsageDataException("performLintFix", err);
-    throw err;
+    if (err.status && err.status == 1) {
```

Comment:
I'd like to understand better why an exception would be thrown if there is not really an error in running the command, but rather it identified a lint error. I am thinking that the command executing successfully should send a success event which includes a data point for the status so that it can report success for "no errors" (e.g. 0) and success for "contains lint errors" (whichever status code). I think if the data point is the status code or a string corresponding to the status code, then we can query it on the other side. What is less than clear is whether we are getting an exception if the command runs OK but there are lint errors that cannot be fixed.

Reply:
With the way I have written it, we send a success event in the event that there are linting errors, so I'm not sure what the issue is specifically.